### ISSUE TITLE
Is this updated for 1.10.2?

### ISSUE THREAD

**Issue by Armored-Dragon** [TiberiumPY/puremine#9]
Is this server updated for 1.10 (protocol 210) so we can use the modern wiki,vg documents and modern clients, or does it still need updating?
 
To do this we should sum up the protocol changes from 1.8 to 1.9 and from 1.9 to 1.10 (possibly snapshots too), and implement change by change.

**Comment by TiberiumPY** [comment]
@Armored-Dragon yes, I want to collaborate, you can freely use all of my code there

**Comment by TiberiumPY** [comment]
@Armored-Dragon do you still interested in this?

**Comment by Armored-Dragon** [comment]
I've moved on to Provenance for the time being, but might still work on this.

**Comment by TiberiumPY** [comment]
@Armored-Dragon what is Provenance?

**Comment by Armored-Dragon** [comment]
https://github.com/Armored-Dragon/ProvenanceX

**Comment by TiberiumPY** [comment]
@Armored-Dragon ok, thanks, good luck with your project!
